corehq/apps/es/case_search.py

```diff
@@ -35,6 +35,7 @@ class CaseSearchES(CaseES):
     @property
     def builtin_filters(self):
         return [
+            case_ids,
```

Comment:
I think this is already defined on `CaseES`, so should already be available here?

Reply:
...I have never noticed this inherits from `CaseES`. Clever. Fixed in ea3830888a4